addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -99,9 +98,7 @@ export class WysiwygAdapterComponent extends ComponentAdapter {
 
             initWysiwyg();
 
-            return () => {
-                this.$editable.off('click.odoo-website-editor', '*');
-            };
+            return () => {};
```

Comment:
It's not required to return a function so I would not do it. Less (strange) code and we could even say "more efficient" :wink: